ITAQA/crawler/emilia_romagna.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Emilia-Romagna data downloader and parser
+"""
+
+from datetime import datetime, timedelta
+from functools import reduce
+import urllib
+import pandas as pd
+
+from ITAQA.core.Pollutants import Pollutant
```

Comment:
On the naming:

- I would change the file Pollutant**s**.py to **Pollutant.py** to keep the usage everywhere of singular nouns
- As an alternative, instead of a new file for this enum, use a single file for all relevant enum definitions under `core/`, for example a single `defs.py` or `enums.py` that hold different enum classes

Both solutions are fine by me, but we can discuss

Reply:
I agree that as it is now it's not the best. I chose Pollutants as the module name so as not to have something like `from Pollutant import Pollutant`, with the module having the same name as the class.
In general I don't like this kind of redundancy in imports, so I quite like your idea of a `defs.py` module. If you're ok with this, I'd go ahead and implement it.

Reply:
Also, package and module names should be lowercase as per PEP 8, so we also have that to fix if we want ;)
I.e. ITAQA should be itaqa, and AirQualityStation.py should be renamed to something else (sensors.py?), and then contain the AirQualityStation class and eventually other classes we might need associated to this, if any.
Sorry for the spin-off.

Reply:
Yes, absolutely! `defs.py` is a proposal, if you have a better name, go along, nevertheless I would leave it there for now, under `core/`. I agree, `from abc import abc` is not a nice solution.

Reply:
On the renaming of the module to be compliant with PEP8, ok! I will do it, thanks for the info!

Reply:
Ok I'll rename Pollutants to defs and leave the rest to you!